source/scripting_v3/GTA/Entities/Vehicles/HandlingData.cs

```diff
@@ -738,6 +738,29 @@ public float PetrolTankVolume
 			}
 		}
 
+		public float PetrolConsumptionRate
```

Comment:
I couldn't find any occurrences of `fPetrolConsumptionRate` in any handling.meta files, but I can find a occurrence of the hashed value `0x80593873` (by joaat func without applying case conversion) of `fPetrolConsumptionRate` in dumped exe. I can confirm the offset is right after `fOilVolume` too.